.circleci/config.yml

```diff
@@ -520,11 +565,13 @@ workflows:
       - barretenberg-js: *yarn_project
       - merkle-tree: *yarn_project
       - p2p: *yarn_project
+      - p2p-bootstrap: *yarn_project
       - noir-contracts: *yarn_project
       - noir-compiler: *yarn_project
       - sequencer-client: *yarn_project
       - types: *yarn_project
       - circuits-js: *yarn_project
+      - rollup-provider: *yarn_project
 
       - e2e-join:
```

Comment:
Add the new jobs to e2e-join requirements, so all unit tests finish before we start the e2e ones

Reply:
Missed that, thanks